Documentation/install/upgrade.rst

```diff
@@ -243,6 +243,18 @@ IMPORTANT: Changes required before upgrading to 1.7.0
 
 * Cilium has bumped the minimal kubernetes version supported to v1.11.0.
 
+* The ``kubernetes.io/cluster-service`` label has been removed from the Cilium
+  `DaemonSet`. Existing users must remove this label from their `DaemonSet`
+  specification to safely upgrade.
```

Comment:
Note that following this steps will restart all cilium-agents in the cluster. (@joestringer can you check if it's all at the same time or if it is one by one like it would be a normal upgrade)?

Reply:
I tried it, it follows the usual rollout strategy (eg max 2 unavailable nodes).